Header header

float64 duty_cycle_left
float64 duty_cycle_right